org.civicrm.bootstrapcivicrm/scss/civicrm/common/_common.scss

```diff
@@ -20,3 +20,4 @@
 @import 'notifications';
 @import 'tooltip';
 @import 'wizard';
+@import 'widget-dashlet';
```

Comment:
Only properties should be sorted alphabetically. Sorting the @import may affect on the dependency